server/app/views/_partials/tracking-lateral.hbs

```
@@ -0,0 +1,17 @@
+<script>
+(function() {
+	window.trackLateralPageView = function() {
+		if (!M.prop('queryParams.noexternals')) {
+			(function(i) {
+				var a = document.location.pathname.replace(/\/+$/,"").split("/").pop(),
+					b = RegExp("wikia_beacon_id=([A-Za-z0-9_-]{10})").exec(document.cookie);
```

From my testing it fails silently. What were the issues you've encountered in the past?